integration_tests/utils.py

```diff
@@ -109,6 +111,12 @@ def wait_for_ipc(path, timeout=40.0):
             )
 
 
+def wait_for_current_block_confirmation(w3):
```

Comment:
This method seems like same as `wait_for_new_blocks(1)`.

Reply:
It is different, sometimes the block number did not increase when the first time to call ` w3.eth.get_block_number()`. The purpose here is not simply wait for one block.

Reply:
I don't see the difference, isn't the code is just waiting for a new block mined?

Reply:
Sometimes, I found deploy contracts thread and call contracts thread will make all transactions included in one block. With this, it make sure they will split it into two or three blocks (1 empty).

But now, I can not find it is necessary anymore, I already deleted the `wait_for_current_block_confirmation`.